### ISSUE TITLE
Installing packages with npm is painfully slow in 6.1.0

### ISSUE THREAD

**Issue by mikr13** [npm/npm#21109]
#### I'm opening this issue because:

  - [ ] npm is crashing.
  - [ ] npm is producing an incorrect install.
  - [ ] npm is doing something I don't understand.
  - [ ] npm is producing incorrect or undesirable behaviour.
  - [x] Other (_see below for feature requests_):

#### What's going wrong?
installing packages with npm stalls and no progress is being shown for much time. I have tried it with many packages and even small ones also but the stalling remains the same. I am using git bash.

### supporting information:

 - `npm -v` prints: '6.1.0'
 - `npm config get registry` prints: https://registry.npmjs.org/
 - Windows, OS X/macOS, or Linux?: Windows 10

**Comment by papandreou** [comment]
Seeing the same here. Disabling the audit feature with `npm install --no-audit` seems to help.

**Comment by mikr13** [comment]
But this is a temporary solution as the audit is a nice feature actually.

**Comment by papandreou** [comment]
Yes, obviously :). Does disabling it alleviate the problem for you too?

**Comment by papandreou** [comment]
After we've switched to `npm install --no-audit` in our CI configs, we're still seeing a high rate of failed builds, essentially:

```
npm install --no-audit
Too long with no output (exceeded 10m0s)
```

Observed on both CircleCI and Travis.

... Suggesting that it's not caused by the audit feature. Maybe it's a registry availability problem or another bug in npm 6.x?

**Comment by DanielRuf** [comment]
Can you check if this is due to infrastructure issues at npmjs.org?
For CI I recommend using `npm ci` and committing the lockfile.

**Comment by papandreou** [comment]
It has certainly happened at times when there were no active issues listed at https://status.npmjs.org/
